src/Microsoft.AspNetCore.OData/Extensions/ODataServiceCollectionExtensions.cs

```diff
@@ -139,5 +139,31 @@ public static IServiceCollection AddODataQueryFilter(this IServiceCollection ser
             services.TryAddEnumerable(ServiceDescriptor.Singleton<IFilterProvider>(new QueryFilterProvider(queryFilter)));
             return services;
         }
+
+        /// <summary>
+        /// Enables support for case insensitive model binding.
+        /// </summary>
+        /// <param name="services">The services collection.</param>
+        public static IServiceCollection EnableODataCaseInsensitiveModelBinding(this IServiceCollection services)
+        {
+            return EnableODataCaseInsensitiveModelBinding(services, new ODataModelBinderSettings { EnableCaseInsensitiveModelBinding = true });
+        }
+
+        /// <summary>
+        /// Enables support for case insensitive model binding.
+        /// </summary>
+        /// <param name="services">The services collection.</param>
+        /// <param name="settings">Instance of <see cref="ODataModelBinderSettings" /></param>
+        public static IServiceCollection EnableODataCaseInsensitiveModelBinding(this IServiceCollection services, ODataModelBinderSettings settings)
```

Comment:
add extension method for "IODataBuilder"

Reply:
#Resolved